perfume/__init__.py

```diff
@@ -57,11 +58,17 @@ def _load(self):
         for name in dir(self):
             method = self.__getattribute__(name)
             try:
```

Comment:
Yep, I usally agree on using EAFP, indeed i am kind of using it, but, the 'try' clause not only catches AttributeErrors because of the access to perfume_route or perfume_args but also masks any AttributeError raised inside the call to self.app.route. In fact, i found myself in that situation while developing the PR: some AttributeError was raised within self.app.route call and i didn't notice. That's why i separated the try-bounded access to the attributes and the call to route(). There is no chance of having race-conditions here to be picky. But in order no to duplicate the acces to the attributes but still puting the call outside the try, how about this?
```python
try:
    route = method.perfume_route
    args = method.perfume_args
except AttributeError:
    continue
self.app.route(route, **args)(method)
```